apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -306,22 +319,37 @@ Keep in mind you are not just generating an example; you are generating an actua
       creator_id: authnUserId,
     });
 
-    await queryAsync(sql.insert_ai_question_generation_prompt, {
-      question_id: saveResults.question_id,
-      prompting_user_id: authnUserId,
-      prompt_type: 'initial',
-      user_prompt: prompt,
-      system_prompt: sysPrompt,
-      response: completion.choices[0].message.content,
-      html: results?.html,
-      python: results?.python,
-      errors,
-      completion,
-      job_sequence_id: serverJob.jobSequenceId,
-    });
+    const promptId = await queryRow(
```

Comment:
We're trying to stick to the convention of singular table name + "_id", so this would be `ai_question_generation_prompt_id`. (And similarly below.)